theme/themes/pastanaga/globals/site.overrides

```diff
@@ -677,9 +683,9 @@ body {
 
   .toolbar {
     position: absolute;
-    z-index: 1;
+    z-index: 2!important;
```

Comment:
The important is not required.

Reply:
Without, its not working....

Reply:
it does:
<img width="1039" alt="Screenshot 2019-03-14 at 14 26 08" src="https://user-images.githubusercontent.com/486927/54360459-2fda7d00-4665-11e9-86b8-a8f4b0c71410.png">